Fix `engine_type_colors`'s description

Dr-helicopter opened the pull request:
in editor>editor settings>Text Editor>Theme, if you hover hover over "Engine Type Color", it has "Vector2", "Vector3"and "Color" as example but these are not Engine types. and their color is controlled by the "Base Type Color" setting. I discovered this while trying to customize my editor a bit more.

<!--
Please target the `master` branch in priority.

Relevant fixes are cherry-picked for stable branches as needed by maintainers.

To speed up the contribution process and avoid CI errors, please set up pre-commit hooks locally:
https://docs.godotengine.org/en/latest/contributing/development/code_style_guidelines.html
-->

AThousandShips commented:
Please restore the translation files, they are handled elsewhere, see [the instructions](https://docs.godotengine.org/en/stable/contributing/documentation/editor_and_docs_localization.html)

Dr-helicopter committed:
Fix engine_type_colors's description

in editor>editor settings>Text Editor>Theme, if you hover hover over "Engine Type Color", it has "Vector2", "Vector3"and "Color" as example but these are not Engine types. and their color is controlled by the "Base Type Color" setting. I discovered this while trying to customize my editor a bit more.

Dr-helicopter commented:
I have restored the translation files to their previous state(and accidentally closed my pr. I apologize for that. its my first pr)

AThousandShips commented on doc/classes/EditorSettings.xml:
```diff
@@ -1395,7 +1395,7 @@
 		</member>
 		<member name="text_editor/theme/highlighting/engine_type_color" type="Color" setter="" getter="">
-			The script editor's engine type color ([Vector2], [Vector3], [Color], ...).
+			The script editor's engine type color ([Node], [Node2D], [Control], ...).
```
```suggestion
			The script editor's engine type color ([Object], [Mesh], [Node], ...).
```
I'd use a few different kinds of types here, using only `Node` types might create confusion that only those are covered

AThousandShips reviewed (approved):
Checked the code and this looks correct, just one small note

Dr-helicopter commented:
yeah, that describes it better

AThousandShips commented:
Please squash your commits into one, see [the interactive rebase](https://docs.godotengine.org/en/latest/contributing/workflow/pr_workflow.html#the-interactive-rebase) for instructions

Dr-helicopter commented:
anything else

AThousandShips commented:
No this should be ready to go

Repiteo commented:
Thanks! Congratulations on your first contribution! 🎉
